pkg/utils/utils.go

```diff
@@ -265,3 +270,69 @@ func GetArgs(command string, frameWork string, frameworkVersion int,
 
 	return args
 }
+
+// Read locators from the file and convert it into array of locator config
+func ExtractLocators(locatorFilePath, flakyTestAlgo string, logger lumber.Logger) ([]core.LocatorConfig, error) {
+	locatorArrTemp := []core.LocatorConfig{}
+	inputLocatorConfigTemp := &core.InputLocatorConfig{}
+
+	if flakyTestAlgo == core.RunningXTimesShuffle {
+		content, err := os.ReadFile(locatorFilePath)
+		if err != nil {
+			logger.Errorf("error when opening file at location %s, %v", locatorFilePath, err)
+			return nil, err
+		}
+
+		err = json.Unmarshal(content, &inputLocatorConfigTemp)
+		if err != nil {
+			logger.Errorf("error during Unmarshal() %v", err)
+			return nil, err
+		}
+		locatorArrTemp = inputLocatorConfigTemp.Locators
+	}
+
+	return locatorArrTemp, nil
+}
+
+// ShuffleLocators shuffles order of elements in locator array
+func ShuffleLocators(locatorArr []core.LocatorConfig, locatorFilePath string, logger lumber.Logger) error {
```

Comment:
locator array is being passed so that we won't have to read every time from the file which would affect execution time.
so I have read from the file once and used it a number of times the execution is run. 
Yes, it should not be exported function. Before the last commit, it was supposed to be exported but that was changed in the last commit, I missed correcting that.